src/main/java/Main.java

```diff
@@ -24,7 +25,8 @@ public static void main(String[] args){
 
 //        // TODO
         User user = new RegularUser("User", "");
-        new playlistView(user);
+        playlistView PlaylistView = new playlistView(user);
```

Comment:
This is still using Users in Main which is a violation of clean architecture I believe since Regular User is an entity. I know it's just temporary to make the Views work but we should probably merge this pull request before the one that @rafaelgacesa made since his PR deals with this issue with InMemoryUsers. Merging this pull request first will probably avoid unnecessary merge conflicts, since the other pr will delete this anyways.